tests/python/unittest/test_operator.py

```diff
@@ -4442,6 +4442,18 @@ def test_where_numeric_gradient(shape, same_shape):
             condition_np, x_np, y_np = get_forward_inputs_condition_vector(shape)
         check_numeric_gradient(where_sym, [condition_np, x_np, y_np], grad_nodes=['x', 'y'])
 
+    def test_invalid_shape():
+        condition = mx.sym.Variable('condition')
+        x = mx.sym.Variable('x')
+        y = mx.sym.Variable('y')
+        where_sym = mx.sym.where(condition, x, y)
```

Comment:
can you add similar tests for mx.nd.where

Reply:
@anirudh2290 Added test for mx.nd.where.